recipes/musclex/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "musclex" %}
+{% set version = "1.23.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/musclex-{{ version }}.tar.gz
+  sha256: 239bcb1accefae4d1bafb8837fec79996d0d7cdeb56bac3590c44a43b948ad00
+
+build:
+  entry_points:
+    - musclex=musclex.main:main
+    - musclex-launcher=musclex.launcher:LauncherForm.main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python 3.10.*
+    - pip
+  run:
+    - python 3.10.*
+    - pip
+    - wheel
+    - numpy <1.24,>=1.18
+    - cython <=0.29.33
+    - scikit-image
+    - scikit-learn
+    - openpyxl <=3.1.1
+    - pyopencl <=2022.3.1
+    - tifffile <=2023.2.28
+    - pandas <=1.5.3
+    - lmfit <=1.1.0
+    - configparser <=5.3.0
+    - pillow <=9.4.0
+    - fabio <=2022.12.1
+    - peakutils <=1.3.4
+    - h5py <=3.8.0
+    - hdf5plugin <=4.1.1
+    - scipy <=1.10.1
+    - matplotlib-base <3.7,>=3.6.3
+    - opencv-python-headless <=4.7.0.72
+    - pyqt <=5.15.9
+    - pyfai <=2023.2.0
+    - distro <=1.8.0
+    - numba <=0.57.1
+    - fisx <=1.2.1
+    - future <=0.18.3
+    - pymca 
+
+test:
+  imports:
+    - AppImageBuilder
+    - dev_docs
+    - docs
+    - hooks
```

Comment:
These should not be installed. This sounds like there is an issue in upstream's call to find_packages.